src/Components/CreateImageWizardV2/validators.ts

```diff
@@ -31,7 +31,9 @@ export const isGcpEmailValid = (gcpShareWithAccount: string | undefined) => {
 };
 
 export const isBlueprintNameValid = (blueprintName: string) =>
-  blueprintName.length > 0 && blueprintName.length <= 100;
+  blueprintName.length >= 2 &&
+  blueprintName.length <= 100 &&
+  /\w+/.test(blueprintName);
```

Comment:
@regexowl I've changed this to only require at least two characters, I've realized we do not care about cool names, as long as we can transpose those to machine friendly image name.
I've filed https://github.com/osbuild/image-builder/pull/1093 to make the transposition happen :)

Reply:
I see, nice! Should this be blocked until we merge https://github.com/osbuild/image-builder/pull/1093 then? To avoid allowing cool names that will break the image during build?

Reply:
well we allow them today, so this would be improvement to today's situation, but sure, I'm all for fixing it properly on backend first ;)

Reply:
Ah yeah, you're completely right. I got the validation mixed up, sorry about that :sweat_smile: I think we can merge then! :)